modules/xfem/doc/content/modules/xfem/index.md

```diff
@@ -17,13 +17,12 @@ Links on the objects' names navigate to an individual detailed page for the obje
        style=width:100%;float:right;padding-top:2.7%;
 
 !row!
-
+!col small=12 medium=6 large=6
```

Comment:
!row must contain !columns
https://mooseframework.org/python/MooseDocs/extensions/layout.html

Here is an example, it needs some more explanation. The numbers for small, etc. come from this: https://materializecss.com/grid.html